pyprocessing/pyprocessing.py

```diff
@@ -27,7 +27,10 @@ class PPNamespace(dict):
         'window_offset': (2, 2),
         'window_title': 'pyprocessing',
         'window_resizable': (False, False),
-        'window_icon': PImage(Image.open(resources.open_binary(images, 'icon16x16.png')))
+        'window_icon': PImage(Image.open(resources.open_binary(images, 'icon16x16.png'))),
+        'cap': 'round',
+        'join': 'miter',
+        'thickness': 1,
```

Comment:
be more specific with the variable name. `thickness` could refer to all kinds of thicknesses, so specify that it's the `stroke_thickness` instead.

Same goes for cap, and join, which can be ambiguous down the line.